docker-compose.l3.yml

```diff
@@ -123,7 +123,7 @@ services:
       - -c
       - "/script/start-zkevm-chain-rpcd.sh"
     deploy:
-      replicas: ${ZKEVM_CHAIN_INSTANCES_NUM}
+      replicas: ${ZKEVM_CHAIN_INSTANCES_NUM:-1}
```

Comment:
emm shall we use `1` here as the default `replicas` value?

Reply:
yes it is 1 right? or i might have the syntax wrong if chatGPT lied.

Reply:
like if it is not set, then it will be 1, i believe.

Reply:
Ah yeah `:-` is the way to set default value in docker compose, i thought its `:`...